codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/protocols/serialize/XmlBindingTraitSerializerGenerator.kt

```diff
@@ -191,10 +191,13 @@ class XmlBindingTraitSerializerGenerator(
         }
     }
 
-    override fun serverOutputSerializer(operationShape: OperationShape): RuntimeType {
+    override fun serverOutputSerializer(operationShape: OperationShape): RuntimeType? {
         val fnName = symbolProvider.serializeFunctionName(operationShape)
         val outputShape = operationShape.outputShape(model)
-        val xmlMembers = operationShape.operationXmlMembers()
+        val xmlMembers = operationShape.serverOutputXmlMembers()
+        if (!xmlMembers.isNotEmpty()) {
```

Comment:
```suggestion
        if (xmlMembers.isEmpty()) {
```

Reply:
@rcoh maybe you can help me out here - I get these compilation errors for calling isEmpty -

```
e: /home/runner/work/smithy-rs/smithy-rs/codegen/src
  /main/kotlin/software/amazon/smithy/rust/codegen/smithy
  /protocols/serialize/XmlBindingTraitSerializerGenerator.kt: (110, 24):

Unresolved reference. None of the following candidates is applicable
  because of receiver type mismatch:

public inline fun <T> Array<out TypeVariable(T)>.isEmpty(): Boolean defined in kotlin.collections
public inline fun BooleanArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun ByteArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun CharArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun CharSequence.isEmpty(): Boolean defined in kotlin.text
public inline fun DoubleArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun FloatArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun IntArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun LongArray.isEmpty(): Boolean defined in kotlin.collections
public inline fun ShortArray.isEmpty(): Boolean defined in kotlin.collections
```

Am I missing an import or something?

Reply:
@rcoh Thanks! added.